CODEOWNERS

```diff
@@ -1 +0,0 @@
-* @puppetlabs/nebula-community
```

Comment:
I believe we are required by opensource law at Puppet to include this file/line, FYI. I'll create a new group called `relay-community` and put people in it.

Reply:
sure :)